mumax3c/scripts/energy.py

```diff
@@ -1,13 +1,14 @@
 import micromagneticmodel as mm
 import numpy as np
+import ubermagutil.typesystem as ts
 
 import mumax3c as mc
 
 
 def energy_script(system):
     mx3 = ""
-    for term in system.energy:
-        mx3 += globals()[f"{term.name}_script"](system)
+    for term in system.energy:  # TODO: different terms of same class not allowed
```

Comment:
We need to check that there is only one term per type and raise an error (maybe `RuntimeError`) otherwise.

Reply:
Thank you for reminding me this. I have made the changes in 576dc5f, please have a look.

Reply:
Looks good